.nvmrc

```diff
@@ -1 +1 @@
-14
+15
```

Comment:
Is there any impact of upgrading this? Why is it needed?

Reply:
Yes. Because npm 7 is only available by default with Node 15+. That's why the PR description says Node 15 needs to be installed.

(npm 7 is the version that supports workspaces)